src/AppWrapper.js

```diff
@@ -22,6 +22,7 @@ const AppWrapper = () => {
                 api.get('userSettings', { useFallback: false }),
                 d2.system.settings.all(),
                 api.get('2fa/enabled'),
+                api.get('configuration/twoFactorMethods'),
```

Comment:
for me this crashes the app in v41? Because the rejection causes the whole promise.all to be rejected